cdp/wallet.py

```diff
@@ -228,29 +259,48 @@ def list(cls) -> Iterator["Wallet"]:
             page = response.next_page
 
     @classmethod
-    def import_data(cls, data: WalletData) -> "Wallet":
-        """Import a wallet from previously exported wallet data.
+    def import_data(cls, data: WalletData | MnemonicSeedPhrase) -> "Wallet":
+        """Import a wallet from previously exported wallet data or a mnemonic seed phrase.
```

Comment:
This is importing a wallet from previously exported wallet data or "creating a new wallet" from a mnemonic seed phrase.

It feels like we should split out the "load" from the "create" flows here? Or is the intent to lazily create the wallet (if one does not exist)? In that world we'd need to be able to check if a wallet already exists when providing the Mnemonic Seed Phrase as well?

@derek-cb @John-peterson-coinbase what are y'alls thoughts here?

Reply:
**Edit**: See that this is a 2-step plan:
Move WalletData out of the import() method and into its own method (ie, load())

Seems reasonable, although I feel like `import` would make more sense than `import_data` for that behavior of taking a mnemonic seed phrase or seed hex, and creating a wallet?

Reply:
ah yes good point. while this is a two-step plan, as you mentioned it doesn't quite make sense in the context of the Python SDK since the existing method is called "import_data". perhaps i can rename this method to "import", and create a new method called "import_data" (that we will remove in a future breaking release) that points to "import". @alex-stone what do you think?

Reply:
That seems reasonable!

Reply:
@alex-stone committed now